ui/src/SpaceDashboard/SpaceDashboard.scss

```diff
@@ -34,6 +34,13 @@ $spaceTileWidth: 400px;
   grid-gap: 32px;
   margin-top: 50px;
   padding: 88px;
+
+  & > * {
```

Comment:
Also curious what this is for

Reply:
We got additional feedback from Sharon where she pointed out that the styling that you and I added to the space tiles wasn't consistently applied to the create new space button; this forces any tile added to the grid to all be the same position and size.